commerce_coordinator/apps/commercetools/clients.py

```diff
@@ -388,17 +393,45 @@ def update_return_payment_state_after_successful_refund(self, order_id: str,
         try:
             logger.info(f"[CommercetoolsAPIClient] - Updating payment state for return "
                         f"with id {return_line_item_return_id} to '{ReturnPaymentState.REFUNDED}'.")
-
             return_payment_state_action = OrderSetReturnPaymentStateAction(
                 return_item_id=return_line_item_return_id,
                 payment_state=ReturnPaymentState.REFUNDED
             )
+            if not payment_intent_id:
```

Comment:
After some more fiddling around and testing, I'm not sure if this is needed. Would there be a scenario when the payment_intent_id coming down the pipeline is empty? And it would be caught by the try/catch block where a CT error would be returned `[2024-09-03 23:59:58,943: ERROR/ForkPoolWorker-8] [CommercetoolsError] Unable to update ReturnPaymentState of order sample_order_number - Correlation ID: projects-sample, Details: [{'code': 'ResourceNotFound', 'message': "The Resource with key 'None' was not found."}]`